api/v1alpha1/flowcollector_types.go

```diff
@@ -196,9 +196,11 @@ type FlowCollectorEBPF struct {
 	// logLevel defines the log level for the NetObserv eBPF Agent
 	LogLevel string `json:"logLevel,omitempty"`
 
-	// privileged mode for the eBPF Agent container. If false, the operator will add the following
-	// capabilities to the container, to enable its correct operation:
-	// BPF, PERFMON, NET_ADMIN, SYS_RESOURCE.
+	// privileged mode for the eBPF Agent container. In general this setting can be ignored or set to false:
```

Comment:
@mariomac can you take a look at this description in particular?

Reply:
Much clearer, thanks for adding context to this!